src/chicletSlicer.ts

```diff
@@ -696,6 +697,13 @@ module powerbi.extensibility.visual {
                 data.slicerDataPoints = data.slicerDataPoints.filter(x => x.selectable);
             }
 
+            // Validate image URL
+            data.slicerDataPoints.forEach(x => {
+                if (!ValidationHelper.isImageUrlAllowed(x.imageURL)) {
```

Comment:
I'm afraid that might break reports that use links w/o extension like Google Drive.